src/plugins/cordova-plugin-battery-status/sim-host.js

```diff
@@ -1,14 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
+var telemetry = require('../../client-common/client-telemetry-helper');
```

Comment:
Instead of requiring the module using relative paths, you can define the client-telemetry-helper module to be available in the browserify search path,
```
var telemetry = require('telemetry-helper');
```
So, if this changes the location, you do not need to update all of the plugins simulation files, and you can provide a better infrastructure for developers that wants to develop simulation for their plugins and use telemetry too.

To do this, go to `src/server/sim-files.js`, at the function `getBrowserifySearchPaths`, add the client-common folder to the browserify search path for sim-host,

```
_browserifySearchPaths[simHost] = [dirs.modules[simHost], dirs.modules['common'], dirs.thirdParty]; // here you should add it
```